### DISCUSSION TITLE
How can I test your code?

### DISCUSSION THREAD

**Discussion by MikeDBT** [TENK-DAO/tenk#94] [Q&A]
Hi Tenk Dao Team
I'm Mike I'm artis
I just start learn about coding.I just done NFT-tutorial-frontend with this Video https://www.youtube.com/watch?v=Rt_cmIq12lo
Now I try to clone your code and run yarn and yarn build but I don't know what I have to do next ... 
Can you help me or suggest me the way to complete My goal is make web mint NFT with Paras 
I really want to learn about that if you have courses How to do Please let me know :)
I really appreciate Tenk Dao Team for did open source

**Comment by willemneal** [comment]
Hey Mike, we plan on adding more tutorials and walkthroughs for using the repo, but have been too busy.

1. To test the code, `yarn test`
2. Edit `script/deploy.ts` to make sure the contract metadata is correct.
3. To deploy use `yarn deploy:testnet --accountId <testnetaccount.testnet>`, or `yarn deploy --accountId <mainnetaccount.near>`.  

Once deployed you can use https://tenk-dao.github.io/tenk/ to interact with your contract.

There is also look at https://github.com/TENK-DAO/frontend-starter for generating a frontend for minting.

Again I wish we had more information, but likely won't have time until later this summer.

**Reply by MikeDBT** [reply]
Thank you so much Willemneal

**Comment by MikeDBT** [comment]
Hi Willemneal

I have question if I already mint NFT to my wallet(main net) I need to know How my NFT show on Paras ?

**Reply by willemneal** [reply]
You need to ask paras to whitelist your contract so that you can add your tokens to their site.  Check out their discord or telegram.
